insights/parsers/nova_user_ids.py

```diff
@@ -0,0 +1,72 @@
+"""
+Get ``uid`` of user ``nova`` and ``nova_migration``
+===================================================
+
+The parser class in this module uses base parser class
+``CommandParser`` to get uids of the user ``nova`` and
+``nova_migration``.
+
+Parsers included in this module are:
+
+NovaUID - command ``id -u nova``
+--------------------------------
+
+NovaMigrationUID - command ``id -u nova_migration``
+---------------------------------------------------
+"""
+from insights import CommandParser, parser
+from insights.parsers import SkipException
+from insights.specs import Specs
+
+
+@parser(Specs.nova_uid)
+class NovaUID(CommandParser):
+    '''Parse output of ``id -u nova`` and get the ``uid`` (int).
+
+    Typical output of the ``id -u nova`` command is::
+
+        162
+
+    However the id number may vary.
+
+    Examples:
+
+        >>> nova_uid.data
+        162
+
+    Attributes:
+        data: ``int`` if 'nova user' exist otherwise ``None``.
+
+    Raises:
+        SkipException: If 'nova' user not found or output is empty.
+    '''
+    def parse_content(self, content):
+        self.data = None
+        if not content or (len(content) == 1 and "no such user" in content[0]):
+            raise SkipException()
+        elif len(content) == 1 and ("no such user" not in content[0]):
+            self.data = int(content[0])
```

Comment:
Perhaps it would be useful to have an `else` case that raises a `ParserException` here.  It would avoid a situation where the `id` command changes to have multiple lines of output and `self.data` is `None`. For example `len(content) > 1`.  `ParseException` would be appropriate for the unexpected condition where you expect only 1 line of output and you get multiple lines of output.

Reply:
@bfahr I included `ParseException` in case of multi-line output. I think this is a bit of over testing at the moment(unless `id` command actually has multi-line output).

Let me know if the changes look ok.